test/core/util/run_with_poller.sh

```diff
@@ -16,4 +16,17 @@
 set -ex
 export GRPC_POLL_STRATEGY=$1
 shift
+blackhole_address_setting="$1"
+if [ "$blackhole_address_setting" = "can_create" ]; then
```

Comment:
Btw, with the current state of this change, you'd only be able to set the blackhole option  for tests thats that are set as `uses_poller=True` which is a strange limitation and it feels dirty.